backend/tracim_backend/lib/core/content.py

```diff
@@ -975,7 +976,7 @@ def get_full_pdf_preview_path(self, revision_id: int, file_extension: str) -> st
             pdf_preview_path = self.preview_manager.get_pdf_preview(file_path, file_ext=file_extension)  # nopep8
         except UnavailablePreviewType as exc:
             raise TracimUnavailablePreviewType() from exc
-        except UnsupportedMimeType as exc:
+        except Exception as exc:
```

Comment:
I suggest to continue to except `UnsupportedMimeType` (and raise UnavailablePreview with same message ) and except Exception (and raise UnavailablePreview with message about unkown error).

Reply:
done